tembo-operator/yaml/sample-dedicated-networking.yaml

```diff
@@ -0,0 +1,11 @@
+apiVersion: coredb.io/v1alpha1
+kind: CoreDB
+metadata:
+  name: sample-dedicated
```

Comment:
here is the place to set as 30 character name

Reply:
Added the 30 character name in the integration tests. Skipping this